packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -46,6 +46,26 @@ export const asteroid = (model: ModelData): string[] => [
 model = BaseModel.from_pretrained("${model.id}")`,
 ];
 
+export const audioseal = (model: ModelData): string[] => {
+	const watermarkSnippet = `# Watermark Generator
+	from audioseal import AudioSeal
```

Comment:
I think you need to remove the initial space
```suggestion
from audioseal import AudioSeal
```